java/src/main/java/com/genexus/internet/HttpContext.java

```diff
@@ -486,9 +487,22 @@ public void AddStyleSheetFile(String styleSheet)
 	}
 	public void AddStyleSheetFile(String styleSheet, String urlBuildNumber)
 	{
+		urlBuildNumber = getURLBuildNumber(styleSheet, urlBuildNumber);
 		AddStyleSheetFile(styleSheet, urlBuildNumber, false);
 	}
 
+	private String getURLBuildNumber(String styleSheet, String urlBuildNumber)
+	{
+		if(urlBuildNumber.isEmpty() && !((styleSheet.startsWith("http:")) || (styleSheet.startsWith("https:"))))
```

Comment:
How about using `GXutil.isAbsoluteURL()` method to check if the URL is absolute?